config/locales/ru/admin.yml

```diff
@@ -67,27 +67,27 @@ ru:
       draft: "%{page} черновик"
 
     details:
-      original_language: "Original Language" # TODO: Translate
+      original_language: "Исходный язык" 
       translated_languages: "Языки Перевода"
       priority: "Приоритет На Странице Вдохновение"
       owned_by: "Принадлежит %{owner}"
-      first_published_at: "Впервые опубликован %{datetime}"
+      first_published_at: "Впервые опубликовано %{datetime}"
       vertical_video: "Вертикальное Видео"
       horizontal_video: "Горизонтальное Видео"
       featured_video: "Видео Статьи" # The video which will be shown on an article preview
       featured_date: "Дата Статьи" # The date which will be shown on an article preview
       role: "Роль На Сайте"
-      is_published: "This %{page} is currently visible to the public." # TODO: Translate
-      is_unpublished: "This %{page} is currently not visible to the public." # TODO: Translate
+      is_published: "Эта страница видна пользователям." 
+      is_unpublished: "Эта страница не видна пользователям." 
```

Comment:
We shouldn't remove `%{page}` when it is not necessary to do so. In this case it will always be singular.

Reply:
There's an issue with genders here. "This" would be a different word for "article" and "subtle system node" for example.